src/current/molt/molt-replicator.md

```diff
@@ -0,0 +1,693 @@
+|---------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|
+| `source_lag_seconds`                  | **CockroachDB sources only:** Time between when an incoming resolved MVCC timestamp originated on the source CockroachDB cluster and when it was received by Replicator. |
+| `target_lag_seconds`                  | **CockroachDB sources only:** End-to-end lag from when an incoming resolved MVCC timestamp originated on the source CockroachDB to when all data changes up to that timestamp were written to the target database. |
+| `source_lag_seconds_histogram`        | **CockroachDB sources:** Same as `source_lag_seconds` but stored as a histogram for analyzing distributions over time.<br>**Non-CockroachDB sources:** Time between when a source transaction is committed and when its COMMIT transaction log arrives at Replicator. |
+| `target_lag_seconds_histogram`        | **CockroachDB sources:** Same as `target_lag_seconds` but stored as a histogram for analyzing distributions over time.<br>**Non-CockroachDB sources:** End-to-end lag from when a source transaction is committed to when its changes are fully written to the target CockroachDB. |
+| `replicator_applier_mutations_staged` | Number of mutations that have been staged for application to the target database.                                          |
+| `replicator_applier_mutations_applied` | Number of mutations that have been successfully applied to the target database.                                           |
+
+You can use the [Replicator Grafana dashboard](https://replicator.cockroachdb.com/replicator_grafana_dashboard.json) to visualize these metrics. For Oracle-specific metrics, import [this Oracle Grafana dashboard](https://replicator.cockroachdb.com/replicator_oracle_grafana_dashboard.json).
+
+To check MOLT Replicator health when metrics are enabled, run `curl http://localhost:30005/_/healthz` (replacing the port with your `--metricsAddr` value). This returns a status code of `200` if Replicator is running.
+
+### Logging
+
+By default, MOLT Replicator writes two streams of logs: operational logs to `stdout` (including `warning`, `info`, `trace`, and some errors) and final errors to `stderr`.
+
+Redirect both streams properly to ensure all logs are captured for troubleshooting:
+
+{% include_cached copy-clipboard.html %}
+~~~shell
+# Merge both streams to console
+./replicator ... 2>&1
+
+# Redirect both streams to a file
+./replicator ... > output.log 2>&1
+
+# Merge streams to console while saving to file
+./replicator > >(tee replicator.log) 2>&1
+
+# Use logDestination flag to write all logs to a file
+./replicator --logDestination replicator.log ...
+~~~
+
+Enable debug logging with `--verbose`. Pay close attention to warning- and error-level logs, as these indicate when Replicator is misbehaving.
```

Comment:
Can also note that if they want even more granularity and system insights, they can do `-vv` for trace.